bridgesample/build.gradle

```diff
@@ -41,5 +41,6 @@ dependencies {
 
     // Support Library
     implementation "com.android.support:appcompat-v7:$supportLibraryVersion"
+    implementation "com.android.support:design:$supportLibraryVersion"
```

Comment:
No Jet Pack, you slacker

Reply:
Right, that would make it incompatible with apps that don't use Jetpack, so I have to keep it this way for now.